test/e2e/openshift.test.ts

```diff
@@ -0,0 +1,172 @@
+      try {
+        const token = await helper.getAccessToken(PLATFORM)
+        process.env.CHE_ACCESS_TOKEN = token
+        console.log(token)
+      } catch (error) {
+        console.log(error)
+      }
+    })
+})
+
+describe('Export CA certificate', () => {
+  it('Export CA certificate', async () => {
+    const command = `${binChectl} cacert:export`
+
+    const { exitCode, stdout, stderr } = await execa(command, { timeout: 30000, shell: true })
+
+    expect(exitCode).equal(0)
+    console.log(stdout)
+
+    if (exitCode !== 0) {
+      console.log(stderr)
+    }
+  })
+})
+
+describe('Workspace creation, list, start, inject, delete. Support stop and delete commands for Eclipse Che server', () => {
+  describe('Create Workspace', () => {
+    test
+      .stdout({ print: true })
+      .stderr({ print: true })
+      .command(['workspace:create', '--devfile=test/e2e/util/devfile-example.yaml'])
+      .exit(0)
+      .it('Create a workspace and wait to be started')
+  })
+
+  describe('Start Workspace', () => {
+    it('Start a workspace using execa library', async () => {
+      const workspaceId = await helper.getWorkspaceId(PLATFORM)
+      const command = `${binChectl} workspace:start ${workspaceId}`
+
+      const { exitCode, stdout, stderr } = await execa(command, { timeout: 30000, shell: true })
+
+      expect(exitCode).equal(0)
+      console.log(stdout)
+
+      await helper.SleepTests(200000)
+      if (exitCode !== 0) {
+        console.log(stderr)
+      }
+
+      const workspaceStatus = await helper.getWorkspaceStatus(PLATFORM)
+
+      expect(workspaceStatus).to.contain('RUNNING')
+    })
+  })
+
+  describe('Inject kubeconfig to workspaces', () => {
+    it('Inject kubeconfig to workspaces', async () => {
+      const command = `${binChectl} workspace:inject --kubeconfig`
+
+      const { exitCode, stdout, stderr } = await execa(command, { timeout: 30000, shell: true })
+
+      expect(exitCode).equal(0)
+      console.log(stdout)
+
+      if (exitCode !== 0) {
+        console.log(stderr)
+      }
+    })
+  })
+
+  describe('List Workspace', () => {
+    test
+      .stdout({ print: true })
+      .stderr({ print: true })
+      .command(['workspace:list'])
+      .it('List workspaces')
+  })
+
+  describe('Stop Workspace', () => {
+    it('Stop a workspace using execa library', async () => {
+      const workspaceId = await helper.getWorkspaceId(PLATFORM)
+      const command = `${binChectl} workspace:stop ${workspaceId}`
+
+      const { exitCode, stdout, stderr } = await execa(command, { timeout: 30000, shell: true })
+      expect(exitCode).equal(0)
+
+      console.log(stdout)
+
+      if (exitCode !== 0) {
+        console.log(stderr)
+      }
+    })
+  })
+
+  describe('Delete Workspace', () => {
+    it('Delete a workspace using execa library', async () => {
+      const workspaceId = await helper.getWorkspaceId(PLATFORM)
+      const token = await helper.getAccessToken(PLATFORM)
```

Comment:
I thought we put token to env process.env.CHE_ACCESS_TOKEN in the test "Obtain access_token from keycloak and set it like environment variable."

Reply:
I discovered sometimes token life is not enough for all test.